pgcli/main.py

```diff
@@ -386,65 +450,17 @@ def run_cli(self):
                 # Initialize default metaquery in case execution fails
                 query = MetaQuery(query=document.text, successful=False)
 
-                try:
-                    output, query = self._evaluate_command(document.text)
-                except KeyboardInterrupt:
-                    # Restart connection to the database
-                    self.pgexecute.connect()
-                    logger.debug("cancelled query, sql: %r", document.text)
-                    click.secho("cancelled query", err=True, fg='red')
-                except NotImplementedError:
-                    click.secho('Not Yet Implemented.', fg="yellow")
-                except OperationalError as e:
-                    if ('server closed the connection'
-                            in utf8tounicode(e.args[0])):
-                        self._handle_server_closed_connection()
-                    else:
-                        logger.error("sql: %r, error: %r", document.text, e)
-                        logger.error("traceback: %r", traceback.format_exc())
-                        click.secho(str(e), err=True, fg='red')
-                except Exception as e:
-                    logger.error("sql: %r, error: %r", document.text, e)
-                    logger.error("traceback: %r", traceback.format_exc())
-                    click.secho(str(e), err=True, fg='red')
+                watch_command, timing = special.get_watch_command(document.text)
+                if watch_command:
+                    while watch_command:
+                        try:
+                            query = self.execute_command(watch_command)
+                            watch_command, timing = special.get_watch_command(document.text)
```

Comment:
Do we need this inside the `while` loop? Looks like we're already parsing the watch_command outside the loop and I don't anticipate that it will change.